src/Servers/Kestrel/Core/src/Internal/Http/HttpHeaders.Generated.cs

```diff
@@ -27,7 +27,7 @@ public StringValues HeaderCacheControl
         {
             get
             {
-                StringValues value;
+                StringValues value = default;
```

Comment:
This is a generated file but I don't see changes in the generator. Suggest there should be.

Reply:
Generator change is in `src/Servers/Kestrel/tools/CodeGenerator/KnownHeaders.cs`